components/exercises/chapter-5-oscar/Everything/styles.css

```diff
@@ -0,0 +1,10 @@
+.wrapper:before {
+  content: '';
+  background: url('./assets/sunset.jpg') no-repeat center center/cover;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  opacity: 0.5;
+}
```

Comment:
The problem with adding a style like this is that, if in any other component we add a className of `wrapper`, it will also be affected by this styling. So it doesn't affect only to the component, but is added to the global styling namespace. 

To make sure this styling is encapsuled to this component, can we replace this by some tailwind classes? or if it can't be done with tailwind, can we use inline styles like `<div style={{ opacity: 1 }}>`?

Reply:
I fixed like below, I had to randomly generate images as I couldn't a local file:
<div className='absolute inset-0 bg-cover bg-center'
        style={{
          backgroundImage: `url(https://source.unsplash.com/1600x900/?${backgrounfPhoto})`,
          opacity: 0.5,
        }}
      ></div>